service/src/lib.rs

```diff
@@ -459,6 +460,39 @@ impl Service for SimpleService<sync_wrapper::SyncWrapper<axum::Router>> {
     }
 }
 
+#[cfg(feature = "web-tide")]
+impl Service for SimpleService<sync_wrapper::SyncWrapper<tide::Server<()>>> {
+    fn build(&mut self, factory: &mut dyn Factory) -> Result<(), Error> {
+        if let Some(builder) = self.builder.take() {
+            // We want to build any sqlx pools on the same runtime the client code will run on.
+            // Without this expect to get errors of no tokio reactor being present.
+            let tide = self.runtime.block_on(builder(factory))?;
+
+            self.service = Some(tide);
+        }
+
+        Ok(())
+    }
+
+    fn bind(&mut self, addr: SocketAddr) -> Result<(), error::Error> {
+        let tide = self
+            .service
+            .take()
+            .expect("service has already been bound")
+            .into_inner();
+
+        self.runtime
+            .block_on(async {
+                let mut app = tide::new();
+                app.at("/").nest(tide);
+                app.listen(addr).await
```

Comment:
```suggestion
                tide.listen(addr).await
```
Why is the indirection needed?

Reply:
you're right, thank you